cmd/k8s-operator/operator.go

```diff
@@ -37,13 +37,19 @@ import (
 	"tailscale.com/hostinfo"
 	"tailscale.com/ipn"
 	"tailscale.com/ipn/store/kubestore"
+	tsapi "tailscale.com/k8s-operator/apis/v1alpha1"
 	"tailscale.com/tsnet"
+	"tailscale.com/tstime"
 	"tailscale.com/types/logger"
 	"tailscale.com/version"
 )
 
+// Generate static manifests for deploying Tailscale operator on Kubernetes from the operator's Helm chart.
 //go:generate go run tailscale.com/cmd/k8s-operator/generate
 
+// Generate Connector CustomResourceDefinition yaml from its Go types.
+//go:generate go run  sigs.k8s.io/controller-tools/cmd/controller-gen crd schemapatch:manifests=./deploy/crds output:dir=./deploy/crds paths=../../k8s-operator/apis/...
```

Comment:
I think because of this `go:generate` line you don't need an additional `.github` action. but fine as is.

Reply:
The github action is to test that the generated code is up to date, without it we could still push a PR with yamls out of date wrt the Go types- or you meant something else?